app/services/market_order_processor_service.rb

```diff
@@ -21,6 +21,7 @@ def initialize(orders, server_id, opts)
   def process
     record_data = []
     @orders.each do |order|
+      next unless ["offer", "request"].include?(order["AuctionType"])
```

Comment:
* downcase `order["AuctionType"]`
* use `'` and not `"` so the ruby engine doesnt need to interpolate
* needs tests